src/modules/shared/components/ListNavigation/ListNavigation.module.scss

```diff
@@ -0,0 +1,40 @@
+@use "sass:map";
+@use "styles/abstracts" as *;
+
+.c-list-navigation {
+	&__list {
+		padding: 0;
```

Comment:
Ik heb een `%reset-list` placeholder toegevoegd in de styles onder `abstracts/`
Je kan die hiervoor gebruiken.